### PR TITLE
Medbay Remap: The Enshrinkening

### PR DESCRIPTION
**Pull request by MarinaGryphon**
Substantially remapped Medical.
* Removed the surgical freezer as all its contents could be found elsewhere; O- blood bags were moved to freezers in the OR itself and the second organ freezer was added to storage.
* Added caution stripes to the ORs to mark areas that are off-limits for nonsterile personnel.
* The Operating Rooms now have separate doors.
* One of the ORs now has an observation area.
* Gives paramedics access to the ORs in order to deliver patients. THIS DOES NOT MEAN THAT THEY CAN DO SURGERY.
* Adds an area in Maintenance using the freed-up space.

### CODE REVIEW

**Comment by Snakebittenn** [comment]
I like this at the moment. It reduces unnecessary bloat from the Medical map, and flows better than the current setup.

**Comment by alsoandanswer** [comment]
You forgot to put organ cooler boxes.
I suggest on the empty space on the OR tables. 

Since you ARE removing the freezer entirely. 

Other than that, looks good.

**Comment by alsoandanswer** [comment]
Speaking of which, I'd also suggest a button from inside the ORs from within, like the medical entrance since a gripe I have is patients unable to let themselves out and having to deal with access fuckery.

**Comment by MarinaGryphon** [comment]
> You forgot to put organ cooler boxes.
> I suggest on the empty space on the OR tables.
> 
> Since you ARE removing the freezer entirely.
> 
> Other than that, looks good.
> 
> I guess you could also re-introduce comfy chairs to the left too.

Organ coolers are in storage. There was already one before, and now there's another because one was removed from the freezer.
The comfy chairs were removed because people pointed out that it would just encourage loitering, and also having chairs in the OR wing doesn't make much sense. People should be waiting in the lobby.


> Speaking of which, I'd also suggest a button from inside the ORs from within, like the medical entrance since a gripe I have is patients unable to let themselves out and having to deal with access fuckery.

No one should be letting themselves out of the OR??? You're not cleared for discharge until the surgeon is finished. No patient should be left in the OR unsupervised ever.

**Comment by alsoandanswer** [comment]
Agree. 

However, still think organ coolers should be added

Don't want to run back and forth from storage to surgery just to do a transplant.

**Comment by MarinaGryphon** [comment]
> Agree.
> 
> However, still think organ coolers should be added
> 
> Don't want to run back and forth from storage to surgery just to do a transplant.

you can literally just move the coolers to the OR if you want them there

**Comment by MarinaGryphon** [comment]
`  - maptweak: "Slims Medical maintenance down on the sublevel and psych level."`
`  - maptweak: "Removes the MediExpress system."`
Clarifies that paramedics could deliver supplies as well as deliver patients.

**Comment by MarinaGryphon** [comment]
!review

**Comment by MarinaGryphon** [comment]
image
image

The MediExpress system has been removed. The engineers tasked to do the job appear to have decided only to sever the connection to the main disposals system, leaving the disconnections rather obvious at certain doorways/hazard shutters. I suppose they thought that since no one goes in those areas anymore anyway, it wasn't worth the time needed to pull up the tiles and remove the disposals pipes. Time is money, after all, and no one would notice the difference anyway.

image
image

Additionally, maintenance has been slimmed on the sublevel and psych level due to the removal.

File: code/game/machinery/iv_drip.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	icon = 'icons/obj/iv_drip.dmi'
 	anchored = 0
 	density = 1
+	throwpass = TRUE
```
Comment (NonQueueingMatt) [inline code/game/machinery/iv_drip.dm:6]: need to add this to cl

**Comment by MarinaGryphon** [comment]
Oh shit. I didn't notice the elevator. The table... ehh. I can maybe do that, yeah.

**Comment by MarinaGryphon** [comment]
!wip

**Pull request closed without merging**